tests/functional/test_submitted_charts.py

```diff
@@ -172,11 +185,21 @@ def workflow_is_triggered():
 def submission_tests_run_for_submitted_charts(secrets):
     """submission tests are run for existing charts."""
 
+    # Don't notify on dry runs, default to True
+    dry_run = True
+    if not secrets.dry_run:
+        # Don't notify if not triggerd on PROD_REPO and PROD_BRANCH
+        triggered_branch = os.environ.get("GITHUB_REF").split('/')[-1]
+        triggered_repo = os.environ.get("GITHUB_REPOSITORY")
+        if triggered_repo == PROD_REPO and triggered_branch == PROD_BRANCH:
+            dry_run = False
+
     with TemporaryDirectory(prefix='tci-') as temp_dir:
         owners_table = dict()
         pr_number_list = []
 
         repo = git.Repo(os.getcwd())
+        set_git_username_email(repo, secrets.bot_name, f'{secrets.bot_name}@test.email')
```

Comment:
Found this issue when I test on manual trigger on `zonggen/charts:main`, need to set git username and email